docs/guide.rst

```diff
@@ -320,9 +269,354 @@ attributes ``description`` and ``link``::
+
+    tmt plans create --template mini /plans/smoke
+    tmt plans create --t full /plans/features
+
+When creating many plans, for example when migrating the whole
+test coverage from a different tooling, it might be handy to
+override default template content directly from the command line.
+For this use individual step options such as ``--discover`` and
+provide desired data in the ``yaml`` format::
+
+    tmt plan create /plans/custom --template mini \
+        --discover '{how: "fmf", name: "internal", url: "https://internal/repo"}' \
+        --discover '{how: "fmf", name: "external", url: "https://external/repo"}'
+
+Now it will be no surprise for you that for creating a new story
+the ``tmt story create`` command can be used with the very same
+possibility to choose the right template::
+
+    tmt story create --template full /stories/usability
+
+Sometimes you forget something, or just things may go wrong and
+you need another try. In such case add ``-f`` or ``--force`` to
+quickly overwrite existing files with the right content.
+
+.. _lint:
+
+Lint
+------------------------------------------------------------------
+
+It is easy to introduce a syntax error to one of the fmf files and
+make the whole tree broken. The ``tmt lint`` command performs a
+set of :ref:`lint-checks` which compare the stored metadata
+against the specification and reports anything suspicious:
+
+.. code-block:: shell
+
+    $ tmt lint /tests/execute/basic
+    /tests/execute/basic
+    pass C000 fmf node passes schema validation
+    warn C001 summary should not exceed 50 characters
+    pass T001 correct keys are used
+    pass T002 test script is defined
+    pass T003 directory path is absolute
+    pass T004 test path '/home/psss/git/tmt/tests/execute/basic' does exist
+    skip T005 legacy relevancy not detected
+    skip T006 legacy 'coverage' field not detected
+    skip T007 not a manual test
+    skip T008 not a manual test
+    pass T009 all requirements have type field
+
+There is a broad variety of options to control what checks are
+applied on tests, plans and stories:
+
+.. code-block:: shell
+
+    # Lint everything, everywhere
+    tmt lint
+
+    # Lint just selected plans
+    tmt lint /plans/features
+    tmt plans lint /plans/features
+
+    # Change the set of checks applied - enable some, disable others
+    tmt lint --enable-check T001 --disable-check C002
+
+See the :ref:`lint-checks` page for the list of available checks
+or use the ``--list-checks`` option. For the full list of options,
+see ``tmt lint --help``.
+
+.. code-block:: shell
+
+    # All checks tmt has for tests, plans and stories
+    tmt lint --list-checks
+
+    # All checks tmt has for tests
+    tmt test lint --list-checks
+
+You should run ``tmt lint`` before pushing changes, ideally even
+before you commit your changes. You can set up `pre-commit`__ to
+do it for you. Add to your repository's ``.pre-commit-config.yaml``::
+
+    repos:
```

Comment:
I'd propose sticking to one mode of dealing with "code blocks". There `.. code-block:: shell` directives already used in this file, but also blocks just indented. How about using `.. code-block::` in new text? The fmf snippet below would deserve its own nice and shiny `.. code-block:: yaml` for sure.

Reply:
Good idea with the yaml code block.

Reply:
Fixed in deb1131d.